### PR TITLE
[BugFix] Fix optional imports

### PR DESCRIPTION
**Pull request by vmoens**
## Description

Fix optional broken optional imports

### CODE REVIEW

File: test/test_helpers.py [hunk 1]
```diff
@@ -402,6 +412,7 @@ def test_sac_make(device, gsde, tanh_loc, from_pixels, exploration):
         del proof_environment
 
 
+@pytest.mark.skipif(not _has_hydra, reason="No hydra library found")
```
Comment (apbard) [inline test/test_helpers.py:415]: nit: group the skipif lines either before or after parametrize

File: test/test_loggers.py [hunk 2]
```diff
@@ -7,9 +7,23 @@
...
+try:
+    import mlflow
+
+    _has_mlfow = True
+except ImportError:
+    _has_mlfow = False
+
 from torchrl.trainers.loggers.csv import CSVLogger
-from torchrl.trainers.loggers.mlflow import MLFlowLogger, _has_mlflow
+from torchrl.trainers.loggers.mlflow import MLFlowLogger
```
Comment (apbard) [inline test/test_loggers.py:26]: if removine _has_mlflow here probably worth doing the same for _has_tb and _has_wandb

File: test/test_loggers.py [hunk 3]
```diff
@@ -7,9 +7,23 @@
...
+    import torchvision
+
+    _has_tv = True
+except ImportError:
+    _has_tv = False
+
+try:
+    import mlflow
+
+    _has_mlfow = True
```
Comment (apbard) [inline test/test_loggers.py:21]: typo

File: test/test_loggers.py (hunk 2 above)
Comment (apbard) [reply test/test_loggers.py:26]: I mean, to be consistent I would change also lines 27 and 28

Comment (vmoens) [reply test/test_loggers.py:26]: but how do we get `_has_tb` and `_has_wandb` then? Sorry I probably misunderstood

**Pull request merged**